demo/src/test/java/amazon/AppTest.java

```diff
@@ -1,24 +1,144 @@
 package amazon;
 
-import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.*;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
+import org.testng.annotations.AfterTest;
 import org.testng.annotations.Test;
 
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
 
 public class AppTest {
-   @Test
-    public void mainPage(){
 
-      System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
 
-      WebDriver driver = new ChromeDriver();
-      driver.manage().window().maximize();
 
-      driver.get("https://www.amazon.com/");
+    @Test
+    public void testDeliverTo() {
+
+        System.setProperty("webdriver.chrome.driver",
+                "/Users/test/IdeaProjects/epamATpracticeGroup6/demo/src/test/resources/webdriver/chromedriver"
+        );
+
+        WebDriver driver = new ChromeDriver();
+
+        driver.get("https://www.amazon.com/");
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+        WebElement deliverToButton =
+                driver.findElement(By.xpath("//*[@id=\"glow-ingress-line1\"]"));
+        deliverToButton.click();
+
+        WebElement enterButton = driver.findElement(By.xpath("//*[@id=\"GLUXZipUpdateInput\"]"));
+        enterButton.sendKeys("99801");
+
+        WebElement applyButton =
+                driver.findElement(By.xpath("//*[@id=\"GLUXZipInputSection\"]/div[2]"));
+        applyButton.click();
+
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+        WebElement continueButton =
+                driver.findElement(By.xpath("//div[@class='a-popover-footer']/span[@class='a-button a-column a-button-primary a-button-span4']"));
+        continueButton.click();
+
+        WebElement cityButton =
+                driver.findElement(By.xpath("//*[@id=\"glow-ingress-line2\"]"));
+
+        Assert.assertEquals(cityButton.getText(), "Juneau 99801");
```

Comment:
It is better to add additional text with the explanation. It will be easier to understand what went wrong if the tests failed